components/prize_pool/commons/prize_pool.lua

```diff
@@ -1161,8 +1188,10 @@ function Placement:_displayPlace()
 end
 
 function Placement:getBackground()
-	if Placement.specialStatuses.DQ.active(self.args) then
-		return 'background-color-disqualified'
+	for statusName, status in pairs(Placement.specialStatuses) do
+		if status.active(self.args) and PlacementInfo.getBgClass(statusName:lower())  then
```

Comment:
Do we need `PlacementInfo.getBgClass(statusName:lower())` check? Similar to #1779 removing the check would remove bg's for all special statuses right?

Reply:
correct
question is what behaviour we want, tbh i am fine with both version
for the PR i just tried to keep it close to the current behaviour, but if the change is wanted i am fine with that^^

Reply:
We can do it in a follow up PR, but I makes sense to me for DNP not to have a bronze background (if they happened to be in the 3rd slot)